src/RoboFile/ClientContainer.php

```diff
@@ -42,7 +42,24 @@ public function runContainerTests(
 		$opts['install-test'] = '';
 
 		$this->runContainerTestPreparation($opts);
+
 		$this->runTestSuites($opts);
+
+	}
+
+	public function runContainerTest(
```

Comment:
What's the idea of having 2 functions for the same?  Or just legacy compatibility?